src/containers/Masthead/components/MastheadSearchForm.jsx

```diff
@@ -26,10 +31,33 @@ class MastheadSearchForm extends Component {
     };
     this.handleQueryChange = this.handleQueryChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleUrlPaste = this.handleUrlPaste.bind(this);
   }
 
   handleQueryChange(evt) {
     this.setState({ query: evt.target.value });
+    this.handleUrlPaste(evt.target.value);
+  }
+
+  handleUrlPaste(ndlaUrl) {
```

Comment:
Kunne tenkte meg en test for dette her.

Reply:
@oyvinmar Skal ha lagt til tre test cases nå.